src/renderer/containers/Settings/DolphinSettings.tsx

```diff
@@ -19,6 +19,12 @@ import { SettingItem } from "./SettingItem";
 const { isLinux, isMac } = window.electron.common;
 const log = window.electron.log;
 
+enum ResetType {
+  NONE,
```

Comment:
```suggestion
```
Also, I don't think we need a `NONE` reset type. Instead we could do `React.useState<ResetType | null>(null);` with a `null` initial value.